clients/fides-js/src/docs/fides-options.ts

```diff
@@ -297,4 +297,7 @@ export interface FidesOptions {
    * Defaults to `undefined`.
    */
   fides_modal_default_view?: string;
+
+  // Whether to show the consent modal immediately upon initialization
```

Comment:
Can we update this to `jsdoc` format so it shows up in the Markdown for customers?